storage/migrations/02_oasis_3_emerald.up.sql

```diff
@@ -232,6 +232,32 @@ CREATE TABLE oasis_3.runtime_sdk_balances (
 
 -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
 
+-- Events emitted from the emerald paratime
+CREATE TABLE oasis_3.emerald_events
```

Comment:
As of https://github.com/oasisprotocol/oasis-indexer/pull/296, this should be `evm_events` because it applies to every runtime with an `evm` module (notably, sapphire).

Reply:
Hrm I'm not a fan of `evm_events` since there could be some non-evm runtime events from other modules, ie `accounts.Burn`. Going with `runtime_events` instead.